rest/model/task.go

```diff
@@ -327,6 +335,17 @@ func (at *APITask) BuildFromService(t interface{}) error {
 
 // ToService returns a service layer task using the data from the APITask.
 func (ad *APITask) ToService() (interface{}, error) {
+	if *ad.Status == evergreen.TaskKnownIssue {
```

Comment:
rather than accessing string pointers directly, we typically use utility.FromStringPtr for safety